src/index.js

```diff
@@ -78,21 +63,21 @@ function deleteStacks(stacks, dryRun) {
   });
 }
 
-function listAllStacksWrapper() {
+const listAllStacksWrapper = async () => {
```

Comment:
If we are not awaiting anything in this block then we can ditch the Async keyword

Reply:
that was awaited in line 14, but i've removed the intermediate listAllStacksWrapper that isn't needed now

Reply:
Unless you are awaiting within the code block you don't need the async keyword. You can still use await on line 14 since promises are awaitable